src/System.Text.Json/src/System/Text/Json/Reader/JsonReaderHelper.Unescaping.cs

```diff
@@ -39,6 +39,24 @@ public static string GetUnescapedString(ReadOnlySpan<byte> utf8Source, int idx)
             return utf8String;
         }
 
+        // TODO: Similar to escaping, replace the unescaping logic with publicly shipping APIs from https://github.com/dotnet/corefx/issues/33509
+        public static byte[] GetUnescapedBytes(ReadOnlySpan<byte> utf8Source, int idx)
+        {
+            byte[] unescapedArray = null;
+
+            Span<byte> utf8Unescaped = utf8Source.Length <= JsonConstants.StackallocThreshold ?
```

Comment:
> Updated coverage

The coverage report shows that we are missing some branch here. Do you know which one/why, or is it flagging it incorrectly?

Reply:
It's the `false` branch.

As part of my updates, this check has been removed because now we assert earlier on that the input guid length is at most 216 (the max length for an escaped default format guid)